react/features/analytics/AnalyticsEvents.js

```diff
@@ -1,663 +1,474 @@
-
-/**
- * Clicked on the button to kick a remote participant from the conference.
- *
- * Properties: value: 1, label: participantID
- *
- * @type {String}
- */
-export const REMOTE_VIDEO_MENU_KICK = 'remotevideomenu.kick';
-
-/**
- * Clicked on the button to audio mute a remote participant.
- *
- * Properties: value: 1, label: participantID
- *
- * @type {String}
- */
-export const REMOTE_VIDEO_MENU_MUTE_CLICKED = 'remotevideomenu.mute.clicked';
-
-/**
- * Confirmed the muting of a remote participant.
- *
- * Properties: value: 1, label: participantID
- *
- * @type {String}
- */
-export const REMOTE_VIDEO_MENU_MUTE_CONFIRMED
-    = 'remotevideomenu.mute.confirmed';
-
-/**
- * Clicked on the remote control option in the remote menu.
- *
- * Properties: value: 1, label: participantID
- *
- * Known full event names:
- * remotevideomenu.remotecontrol.stop
- * remotevideomenu.remotecontrol.start
- *
- * @type {String}
- */
-export const REMOTE_VIDEO_MENU_REMOTE_CONTROL_
-    = 'remotevideomenu.remotecontrol';
-
-/**
- * Replacing the currently used track of specified type with a new track of the
- * same type. The event is fired when changing devices.
- *
- * Known full event names:
- * replacetrack.audio
- * replacetrack.video
- *
- * @type {String}
- */
-export const REPLACE_TRACK_ = 'replacetrack';
-
-/**
- * The local participant failed to start receiving high quality video from
- * a remote participant, which is usually initiated by the remote participant
- * being put on large video.
- *
- * @type {String}
- */
-export const SELECT_PARTICIPANT_FAILED = 'selectParticipant.failed';
-
-/**
- * The local participant began using a different audio input device (mic).
- *
- * @type {String}
- */
-export const SETTINGS_CHANGE_DEVICE_AUDIO_IN = 'settings.changeDevice.audioIn';
-
-/**
- * The local participant began using a different audio output device (speaker).
- *
- * @type {String}
- */
-export const SETTINGS_CHANGE_DEVICE_AUDIO_OUT
-    = 'settings.changeDevice.audioOut';
-
-/**
- * The local participant began using a different camera.
- *
- * @type {String}
- */
-export const SETTINGS_CHANGE_DEVICE_VIDEO = 'settings.changeDevice.video';
-
-/**
- * Attempted to start sharing a YouTube video but one is already being shared.
- *
- * @type {String}
- */
-export const SHARED_VIDEO_ALREADY_SHARED = 'sharedvideo.alreadyshared';
-
-/**
- * The local participant's mic was muted automatically during a shared video.
- *
- * @type {String}
- */
-export const SHARED_VIDEO_AUDIO_MUTED = 'sharedvideo.audio.muted';
-
-/**
- * The local participant's mic was unmuted automatically during a shared video.
- *
- * @type {String}
- */
-export const SHARED_VIDEO_AUDIO_UNMUTED = 'sharedvideo.audio.unmuted';
-
-/**
- * Canceled the prompt to enter a YouTube video to share.
- *
- * @type {String}
+ * The constant for the event type 'track'.
+ * TODO: keep these constants in a single place. Can we import them from
+ * lib-jitsi-meet's AnalyticsEvents somehow?
```

Comment:
Yes that should be possible, assuming no circular dependencies get in the way of loading. I'd try importing lib-jitsi-meet, search for `import JitsiMeetJS` for examples, and see if it gets you what you want. At the very least that could answer the question so it doesn't have be left in.

Reply:
Will experiment